### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nplasterer**
playground

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Testing
 
 things
+
+Blah
```
Comment (nplasterer) [inline README.md]: something something something

Comment (nplasterer) [inline README.md]: a comment

**Comment by jessicard** [comment]
PR comment

File: README.md (hunk 1 above)
Comment (jessicard) [inline README.md]: A reply to @nplasterer's comment

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # Testing
 
 things
```
Comment (jessicard) [inline README.md]: A new line comment

File: README.md (hunk 1 above)
Comment (jessicard) [inline README.md]: Another reply

**Comment by jessicard** [comment]
Another PR comment

File: README.md (hunk 2 above)
Comment (jessicard) [inline README.md]: Another reply to a line comment

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+
+Blah
+Blah
+Blah
```
Comment (jessicard) [inline README.md]: Another line comment

File: README.md (hunk 1 above)
Comment (nplasterer) [inline README.md]: hello how are you

Comment (jessicard) [inline README.md]: Hi I'm good how are you!

File: README.md (hunk 2 above)
Comment (nplasterer) [inline README.md]: hi @jessicard

File: README.md (hunk 3 above)
Comment (jessicard) [inline README.md]: Here's another too!

File: README.md (hunk 1 above)
Comment (jessicard) [inline README.md]: bloop

File: README.md (hunk 2 above)
Comment (nplasterer) [inline README.md]: a comment

File: README.md (hunk 1 above)
Comment (jessicard) [inline README.md]: test

File: README.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
 # Testing
 
 things
+
```
Comment (nplasterer) [inline README.md]: this is a a white space 👍

Comment (nplasterer) [inline README.md]: But you should remove it

File: README.md (hunk 2 above)
Comment (nplasterer) [inline README.md]: good job

File: README.md (hunk 1 above)
Comment (nplasterer) [inline README.md]: very nice

File: README.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
...
+Blah
+Blah
```
Comment (nplasterer) [inline README.md]: a comment

**Pull request closed without merging**